### PR TITLE
Add script of RpaaS lint

### PR DESCRIPTION
**Pull request by jianyexi**

### CODE REVIEW

**Review by tiramisuq — APPROVED** [review]

File: src/lintingRpaas.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+export async function runRpaasLint() {
+    const pr = await devOps.createPullRequestProperties(cli.defaultConfig());
+    const configsToProcess = await utils.getConfigFilesChangedInPR(pr);
+    for (const config of configsToProcess) {
+      const store = new UnifiedPipeLineStore(config);
+      if (isRpaasBranch()) {
+        const checker = new ReadmeParser(config);
+        const subType = checker.getGlobalConfigByName("openapi-subtype");
+        if (subType !== "rpaas") {
```
Comment (raych1) [inline src/lintingRpaas.ts:137]: Undefined check?

File: src/lintingRpaas.ts [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        extra: {
+          new: utils.targetHref(utils.getRelativeSwaggerPathToRepo(config)),
+          details: errorMsg,
+        },
+      };
+      return JSON.stringify(result);
+  }
+} 
+
+class UnifiedPipeLineStore {
```
Comment (raych1) [inline src/lintingRpaas.ts:81]: Can we have this extracted as common class to be used by other tool call? e.g. lintDiff call.

File: src/lintingRpaas.ts [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            ? "unset"
+            : `incorrect, expects 'rpaas', but received: ${subType}`;
+          const errorMsg = `For the readme:${config} , the 'openapi-subtype' is ${subMsg}.\n${helpInfo}`;
+
+          console.log(errorMsg);
+          store.appendReadmeErr(errorMsg);
+          process.exitCode = 1;
+          continue;
+        }
+      }
```
Comment (raych1) [inline src/lintingRpaas.ts:149]: Should return directly if readme doesn't have subType defined correctly.

File: src/lintingRpaas.ts (hunk 1 above)
Comment (raych1) [reply src/lintingRpaas.ts:137]: Covered.

File: src/lintingRpaas.ts (hunk 3 above)
Comment (raych1) [reply src/lintingRpaas.ts:149]: It is by calling continue.

**Review by raych1 — APPROVED** [review]

**Pull request merged**